Combinatorial_map/package_info/Combinatorial_map/dependencies

```diff
@@ -14,8 +16,13 @@ Intersections_3
 Interval_support
 Kernel_23
 Kernel_d
+Modifier
 Modular_arithmetic
 Number_types
+Polyhedron
+Polyhedron_IO
 Profiling_tools
+Property_map
 STL_Extension
 Stream_support
+Surface_mesh
```

Comment:
I am glad you managed to make Travis happy.

But the changes of `dependencies` for CMap are still problematic: you cannot depend on `Polyhedron` or `Surface_mesh`.